internal/cli/main.go

```diff
@@ -15,11 +15,46 @@ import (
 	"github.com/golang-migrate/migrate/v4/source"
 )
 
-const defaultTimeFormat = "20060102150405"
+const (
+	defaultTimeFormat = "20060102150405"
+	createUsage       = `create [-ext E] [-dir D] [-seq] [-digits N] [-format] NAME
+	   Create a set of timestamped up/down migrations titled NAME, in directory D with extension E.
+	   Use -seq option to generate sequential up/down migrations with N digits.
+	   Use -format option to specify a Go time format string. Note: migrations with the same time cause "duplicate migration version" error. 
+`
+	gotoUsage = `goto V       Migrate to version V`
+	upUsage   = `up [N]       Apply all or N up migrations`
+	downUsage = `down [N]     Apply all or N down migrations`
+	dropUsage = `drop [-f] [-all]    Drop everything inside database
+	Use -f to bypass confirmation
+	Use -all to apply all down migrations`
+	forceUsage = `force V      Set version V but don't run migration (ignores dirty state)`
+)
+
+func handleSubCmdHelp(help bool, usage string, flagSet *flag.FlagSet) {
+	if help {
+		fmt.Fprintln(os.Stderr, createUsage)
+		flagSet.PrintDefaults()
+		os.Exit(0)
+	}
+}
+func newFlagSetWithHelp(name string, errHandling flag.ErrorHandling) (*flag.FlagSet, *bool) {
```

Comment:
Looks like the `errHandling` param is unused.

Might as  well return `bool` instead  of `*bool`.

Reply:
I guess I have to return a pointer to a bool otherwise the value will always be the one at the creation and I will not get updated after `flag` has finished to parse the arguement.

Reply:
Remove the `errHandling` param since it's always being passed the same value, which [triggers](https://travis-ci.com/github/golang-migrate/migrate/jobs/413888025) the `unparam` linter:
```
internal/cli/main.go:41:38: `newFlagSetWithHelp` - `errHandling` always receives `flag.ExitOnError` (`1`) (unparam)
func newFlagSetWithHelp(name string, errHandling flag.ErrorHandling) (*flag.FlagSet, *bool) {
                                     ^
```